src/main/java/fr/univartois/sonargo/core/ProjectExplorer.java

```diff
@@ -0,0 +1,48 @@
+package fr.univartois.sonargo.core;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.sonar.api.batch.fs.FilePredicates;
+import org.sonar.api.batch.fs.FileSystem;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.batch.sensor.SensorContext;
+
+import fr.univartois.sonargo.core.language.GoLanguage;
+
+public class ProjectExplorer {
+    private SensorContext context;
+
+    public ProjectExplorer(SensorContext context) {
+	this.context = context;
+    }
```

Comment:
I think it would be better here to only have static utility methods providing the context as a parameter.